docs/source/linchpincli_linchpin_invgen.rst

```diff
@@ -1,24 +0,0 @@
-Invgen
```

Comment:
I'm fine with the warnings for the short term, as those docs will return. As for reformatting, this is likely going to be required. If you want to do it while they are commented out, feel free. However, I think the functionality of these will likely change. Probably better to wait and just reformat and reword things once the functionality returns to linchpin.